flake.nix

```diff
@@ -97,6 +97,7 @@
           shellHook = ''
             echo Setting up $S2N_LIBCRYPTO environment from flake.nix...
             export PATH=${openssl_1_1_1}/bin:$PATH
+            alias openssl3=${openssl_3_0}/bin/openssl
             export PS1="[nix $S2N_LIBCRYPTO] $PS1"
```

Comment:
Should we print some message saying this alias is available? It was the first command I tried when I wanted openssl-3, but idk if that's a universal thought :)

Reply:
Maybe print both the commands and the outputs of "openssl version" and "openssl3 version"